apps/web/src/features/auth/email-change-form.tsx

```diff
@@ -0,0 +1,88 @@
+"use client";
+
+import { SubmitButton } from "@/features/form/loading-button";
+import { EmailChangeSchema } from "@/lib/schemas/email-change.schema";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@workspace/ui/components/card";
+import { Input } from "@workspace/ui/components/input";
+import { useState } from "react";
+
+interface EmailChangeFormProps {
+  currentEmail: string;
+  onEmailChange: (formData: FormData) => Promise<void>;
+}
+
+export function EmailChangeForm({ currentEmail, onEmailChange }: EmailChangeFormProps) {
+  const [email, setEmail] = useState("");
+  const [errors, setErrors] = useState<string[]>([]);
+
+  const handleSubmit = async (formData: FormData) => {
+    setErrors([]);
+    
+    // Client-side validation
+    const newEmail = formData.get("email") as string;
+    const validation = EmailChangeSchema.safeParse({ newEmail });
+    
+    if (!validation.success) {
+      const errorMessages = validation.error.issues.map(issue => issue.message);
+      setErrors(errorMessages);
+      return;
+    }
+
+    if (newEmail === currentEmail) {
+      setErrors(["New email must be different from current email"]);
+      return;
+    }
+
+    try {
+      await onEmailChange(formData);
+      setEmail(""); // Clear form on success
+    } catch (error) {
+      // Error handling is done in the server action with serverToast
+      console.error("Error changing email:", error);
+    }
+  };
+
+  return (
+    <form action={handleSubmit}>
+      <Card>
+        <CardHeader>
+          <CardTitle>Email</CardTitle>
+          <CardDescription>
+            Current email: {currentEmail}<br />
+            Change your email address. You will receive a verification email at your current address.
+          </CardDescription>
```

Comment:
remove this . Currnet email should be in the value of the input and we can just edit it, no need carddescriptoin